### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by EricaFinlay**
Hi @KathP1

Would you mind to please do the code review for me for Prac 05?

Many thanks.

Regards,
Erica Finlay

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Do-From_Scratch Exercises
+Emails
+
+Estimate:   2 hours
+Actual:     1 hour 10 min
+
+The solution to this program was consulted on GitHub.
+
+Pseudocode:
```
Comment (KathP1) [inline prac_05/emails.py:13]: I like how you've put the pseudocode here. That makes a lot of sense.

File: prac_05/emails.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+function get_name_from_email(email)
+    initial_string = split email at '@' [0]
+    names = split initial_string at '.'
+    name = join names with space between
+    return name
+"""
+
+
+def main():
+    """Ask the user for their email and name and prints this information."""
```
Comment (KathP1) [inline prac_05/emails.py:41]: Docstrings use imperative voice ('This function will..') so perhaps it should say 'print', rather than 'prints'

**Review by KathP1 — COMMENTED** [review]
Hi Erica
Your code looks neat and succinct to me. I've just added one suggestion around docstrings.
Kind regards
Kath

File: prac_05/emails.py (hunk 2 above)
Comment (EricaFinlay) [reply prac_05/emails.py:41]: Thanks Kath - that was a really clever one to pick up.  Much appreciated!

**Pull request merged**